### PR TITLE
integrate APU converter part

### PR DESCRIPTION
**Pull request by yungchienhsu**
support mobilenet v1 and deeplab v3+ with mobilenet v2 backbone models

### CODE REVIEW

File: mace/python/tools/converter_tool/base_converter.py [hunk 1]
```diff
@@ -477,6 +481,9 @@ def build(self):
                 TransformerRule.TRANSPOSE_CAFFE_RESHAPE_AND_FLATTEN,
                 TransformerRule.FOLD_RESHAPE,
                 TransformerRule.TRANSFORM_MATMUL_TO_FC,
+                # For BtoS -> conv -> StoB -> BN pattern
```
Comment (lee-bin) [inline mace/python/tools/converter_tool/base_converter.py]: StoB -> conv -> BtoS ?

File: mace/python/tools/converter_tool/base_converter.py [hunk 2]
```diff
@@ -510,6 +517,11 @@ def build(self):
                 # Need to be put after SORT_BY_EXECUTION
                 TransformerRule.ADD_QUANTIZE_TENSOR_RANGE,
             ]
+            if self._device == DeviceType.APU.value:
```
Comment (lee-bin) [inline mace/python/tools/converter_tool/base_converter.py]: Place this transform in apu_converter.py?

File: mace/python/tools/converter_tool/base_converter.py [hunk 3]
```diff
@@ -518,6 +530,11 @@ def build(self):
                     TransformerRule.SORT_BY_EXECUTION,
                     TransformerRule.CHECK_QUANTIZE_INFO,
                 ]
+            if self._device == DeviceType.APU.value and self._quantize:
```
Comment (lee-bin) [inline mace/python/tools/converter_tool/base_converter.py]: Place this transform in apu_converter.py?

File: mace/python/tools/converter_tool/transformer.py [hunk 4]
```diff
@@ -1163,7 +1172,7 @@ def transpose_filters(self):
 
             for op in net.op:
                 if (op.type == MaceOp.Conv2D.name or
-                    op.type == MaceOp.Deconv2D.name) and\
+                    op.type == MaceOp.DepthwiseConv2d.name) and\
```
Comment (lee-bin) [inline mace/python/tools/converter_tool/transformer.py]: DepthwiseConv2d of CPU quantize does not need to transpose.

File: mace/python/tools/converter_tool/base_converter.py (hunk 1 above)
Comment (yungchienhsu) [reply mace/python/tools/converter_tool/base_converter.py]: Yes! will fix it

Comment (yungchienhsu) [reply mace/python/tools/converter_tool/base_converter.py]: fixed in commit 689b236

File: mace/python/tools/converter_tool/base_converter.py (hunk 2 above)
Comment (yungchienhsu) [reply mace/python/tools/converter_tool/base_converter.py]: fixed in commit 689b236

File: mace/python/tools/converter_tool/transformer.py (hunk 4 above)
Comment (yungchienhsu) [reply mace/python/tools/converter_tool/transformer.py]: fixed in commit 689b236

**Pull request closed without merging**